### PR TITLE
fix(w3c/headers): allow localizing multiple_alternates

### PR DESCRIPTION
**Pull request by marcoscaceres**
Small refactor to allow localization of "multiple_alternates".

### CODE REVIEW

File: src/w3c/templates/headers.js [hunk 1]
```diff
@@ -25,6 +25,11 @@ const localizationStrings = {
     latest_recommendation: "Latest Recommendation:",
     message_topic: "… message topic …",
     more_details_about_this_doc: "More details about this document",
+    multiple_alternates(plural) {
+      return `This document is also available in ${
+        plural ? "these non-normative formats" : "this non-normative format"
```
Comment (sidvishnoi) [inline src/w3c/templates/headers.js:30]: Maybe we can use the `pluralize` library we've?

**Review by sidvishnoi — APPROVED** [review]
LGTM. One optional suggestion.
(Don't forget to update commit message before merge!)

File: src/w3c/templates/headers.js (hunk 1 above)
Comment (marcoscaceres) [reply src/w3c/templates/headers.js:30]: It can only do single words 😢

Comment (sidvishnoi) [reply src/w3c/templates/headers.js:30]: I was thinking something like:
```js
      const count = plural ? 1 : 0;
      return `This document is also available in ${pluralize('this', count)} non-normative ${pluralize('format', count)}:`;
```
But never mind, not important.

**Pull request merged**